manual/python/index.md

```diff
@@ -142,13 +142,25 @@ Moreover, `addressof` can be used in conjunction with `TTree::Branch` from Pytho
 
 - `SetOwnership(obj, python_owns)`: a Python proxy can either own or not own its internal C++ object. If a Python proxy owns its C++ object and the proxy is being destroyed, the C++ object will be deleted too. This ownership can be modified for a given Python proxy with `SetOwnership`. For example, when calling `SetOwnership(obj, False)`, the user makes sure that the C++ object proxied by `obj` will not be deleted by PyROOT when `obj` is garbage collected; this is useful e.g. if the user knows the deletion will happen in C++ and wants to prevent a double delete. This functionality should be used with care not to produce memory leaks.
 
-## Loading user libraries & jitting
+## Loading user libraries & Just-In-Time compilation (JITting)
 
-PyROOT allows to use *any C++ library* from Python, not only ROOT. This is possible thanks to the automatic and dynamic bindings between Python and C++ that PyROOT provides: without any prior generation of wrappers, at execution time, PyROOT can load and call into C++ code. The following sub-sections cover different supported scenarios for dynamic C++ code invocation.
+PyROOT allows to use *any C++ library* from Python, not only ROOT's libraries. This is possible thanks to the automatic and dynamic bindings between Python and C++ that PyROOT provides.
```

Comment:
I liked the ":" after `PyROOT provides:`, because it explains why it is automatic and dynamic.

Reply:
A British native speaker once told me that one almost never uses colons in English. That's why they disappeared. I can re-introduce them if you want.